cli/src/commands/certificate/poll.rs

```diff
@@ -0,0 +1,21 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+use crate::utils::*;
+
+crate::clap_parser_with_shared_opts_builder!(
+    #[with = config_dir, device, password_stdin]
+    pub struct Args {
+    }
+);
+
+crate::build_main_with_client!(main, poll);
+
+pub async fn poll(_args: Args, client: &StartedClient) -> anyhow::Result<()> {
+    let mut spinner = start_spinner("Poll server for new certificates".into());
```

Comment:
[Here](https://github.com/Scille/parsec-cloud/blob/84d9c33331ff0e6d40cc61707e28fe3db89b51de/cli/src/utils.rs#L432) is an helper

Reply:
I used it initially, however this helper doesn't display the number of new certificates

I think not showing the number of new certificates is fine when the poll is a side task, but for this CLI command it is the main thing so reporting is a good thing

Reply:
The function could easily updated to return the number of certificate